source/lib/message_types.h

```diff
@@ -0,0 +1,12 @@
+#ifndef _MESSAGE_TYPES_H_
+#define _MESSAGE_TYPES_H_
+
+// Create Message Type that takes specific values using Enumerate:
+
+typedef enum {
+    No_Command,
+    PWM_Pause,
+    PWM_Resume
+} message_t;
+
+#endif /* _MESSAGE_TYPES_H_ */
```

Comment:
Someday this might get reorganized into a different file when we get around to implementing whatever CAN protocol we come up with.